stream-transformation-tool-service/pom.xml

```diff
@@ -31,6 +31,12 @@
             <groupId>uk.gov.justice.services</groupId>
             <artifactId>test-utils-core</artifactId>
             <scope>test</scope>
+            <exclusions>
```

Comment:
as above, we'll need to investigate why we need to exclude this version. Also managed dependencies means we'd only need to set it once, hopefully.